.gitignore

```diff
@@ -269,3 +269,6 @@ binaries/
 
 # Do not include these files
 *.zip
+
+# Do not include launch settings
+launchSettings.json
```

Comment:
This is a very specific value to filter. Why is it needed?

Reply:
That file contains the debugging command line arguments that I don't want to include in my commits.